src/components/Comments/Comment.js

```diff
@@ -376,7 +377,7 @@ const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: COLORS.blackgrey,
+    color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been updated. Ensure this change is consistent with the overall design and color scheme of the application.
```diff
- color: COLORS.blackgrey,
+ color: '#828282',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->